meteor/client/lib/lib.tsx

```diff
@@ -53,7 +53,7 @@ export function fetchFrom(input: RequestInfo, init?: RequestInit) {
 
 export function ensureHasTrailingSlash(input: string | null): string | undefined {
 	if (input) {
-		return input.substr(-1) === '/' ? input : input + '/'
+		return input.slice(-1) === '/' ? input : input + '/'
```

Comment:
The entire function body (including condition) can also be condensed into:
```typescript
return input?.replace(/\/*$/, '/')
```

If that is not desirable, then instead of using slice, you can refactor it to: 
```typescript
return input.endsWith('/') ? input : input + '/'
```